Update weekly call ID

jkflying opened the pull request:
This was out of date, update to the latest version

jkflying committed:
Update weekly call ID

This was out of date, update to the latest version

dagar reviewed (changes requested):
Hold on, the EST time is currently wrong.

jkflying committed:
Fix EST

Fix EST and make the times not use AM/PM (which people always confuse at 12PM), but use 24h time instead.

jkflying committed:
And fix PST

Why don't they do all the changes on the same day... sigh
